op-e2e/actions/proofs/operator_fee_test.go

```diff
@@ -213,7 +206,12 @@ func Test_ProgramAction_OperatorFeeConstistency(gt *testing.T) {
 				new(big.Int).Sub(operatorFeeVaultFinalBalance, operatorFeeVaultInitialBalance),
 			)
 		}
-		require.True(t, aliceFinalBalance.Cmp(aliceInitialBalance) < 0, "Alice's balance should decrease")
+
+		if testCfg.Custom == DepositTx {
+			require.True(t, aliceFinalBalance.Cmp(aliceInitialBalance) == 0, "Alice's balance shouldn't have changed")
```

Comment:
```suggestion
			require.True(t, aliceInitialBalance, aliceFinalBalance, "Alice's balance shouldn't have changed")
```
reports more meaningful numbers in case this fails